packages/mgt-components/src/graph/graph.user.ts

```diff
@@ -168,8 +168,11 @@ export async function getUsersForUserIds(graph: IGraph, userIds: string[], searc
         peopleDict[id] = user ? user : null;
       }
     } else if (id !== '') {
-      batch.get(id, `/users/${id}`, ['user.readbasic.all']);
-      notInCache.push(id);
+      if (id.toString() === 'me') {
+        peopleDict[id] = await getMe(graph);
+      } else {
+        batch.get(id, `/users/${id}`, ['user.readbasic.all']);
```

Comment:
should the id be pushed into the cache after this?

Reply:
No. This just collects the api calls to be batch'called. After calling `batch.excuteAll()` then the successfully executed call ids are put in the cache.